package.json

```diff
@@ -29,13 +29,16 @@
     "@solana/wallet-adapter-wallets": "0.19.10",
     "@solana/web3.js": "1.73.0",
     "@svgr/webpack": "^6.5.0",
+    "@types/lodash.debounce": "^4.0.7",
+    "@types/react-syntax-highlighter": "^15.5.6",
```

Comment:
let's move some things to devDependencies?

- @types/*
- build tools (autoprefixer)
- tailwindcss
- stylus
- @svgr/webpack

Reply:
will do